README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+   Copyright 2021 Ericsson AB.
```

Comment:
Is this true though?

See also: https://github.com/eiffel-community/eiffel-repository-template/issues/14

Reply:
I wasn't sure what to put there so I put Ericsson AB.

So, what should I do?